automation_testing/utils/helpers/helper.ts

```diff
@@ -58,4 +58,14 @@ const getDateFormatted = (date: Date) : string => {
 	return dateFormatted;
 }
 
-export { sortDatesAsc, sortDatesDesc, randomIntFromInterval, getMonthFromString, addQuotes, addComma, isACorrentDateRange, randomNumber, getDateFormatted }
\ No newline at end of file
+const generateRandomId = (length: number): string => {
+	let newId = "";
+	const characters = "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789";
+	const charactersLength = characters.length;
+	for (let i = 0; i < length; i++) {
+		newId += characters.charAt(Math.floor(Math.random() * charactersLength));
+	}
+	return newId;
+}
+
+export { sortDatesAsc, sortDatesDesc, randomIntFromInterval, getMonthFromString, addQuotes, addComma, isACorrentDateRange, randomNumber, getDateFormatted, generateRandomId }
```

Comment:
This is just a recommendation, it's up to you. You add the `export` to each helper function to not have a single export with so manny function. I mean something like this:

```
export const generateRandomId = (length: number): string => {
	let newId = "";
	const characters = "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789";
	const charactersLength = characters.length;
	for (let i = 0; i < length; i++) {
		newId += characters.charAt(Math.floor(Math.random() * charactersLength));
	}
	return newId;
}
```

Reply:
I like this idea, I was thinking of ways to rework the file, so I will apply this.